docs/getting_started.md

```diff
@@ -0,0 +1,142 @@
+
+```bash
+(cd $(mktemp -d); curl -L https://github.com/DataDog/dd-trace-php/archive/v0.2.2.tar.gz | tar x --strip-components=1 && phpize && ./configure && make && sudo make install )
+```
+
+### Enabling the extension
+
+After the extensions has been installed, the PHP runtime needs to be configured to use it.
+
+To do that we need to modify `php.ini`. This file’s location depends on how PHP has been built and configured on your system. To find out where yours is, run the following command:
+
+```bash
+$ php --ini
+
+Configuration File (php.ini) Path: /usr/local/etc/php/7.2
+Loaded Configuration File:         /usr/local/etc/php/7.2/php.ini
+Scan for additional .ini files in: /usr/local/etc/php/7.2/conf.d
+Additional .ini files parsed:      /usr/local/etc/php/7.2/conf.d/ext-opcache.ini
+```
+
+Now the only remaining thing is to add following line to the `.ini` file found using above command:
+
+```ini
+extension=ddtrace.so
+```
+
+After restarting the PHP application (eg. `apachectl restart`) the extension should now be enabled. To confirm that it’s being loaded, run:
+
+```bash
+php -m | grep ddtrace
+```
+
+Some systems use different `php.ini` files for command line PHP vs. the web server module, so you may need to confirm both. If you look at the output of `phpinfo()` in a web page, you will see `ddtrace` listed if it’s loaded.
+
+## Enabling automatic tracing
+
+Once the C extension is installed, we need to install the PHP package that provides the actual integrations and framework for sending traces to Datadog.
+
+### Install `datadog/dd-trace` package
+
+```bash
+
+composer config minimum-stability beta # required to install opentracing 1.0.0-beta5
+composer require opentracing/opentracing
+composer require datadog/dd-trace
+```
+
+#### Alternative: Install `datadog/dd-trace` package without changing `minimum-stability`
+
+```bash
+composer require datadog/dd-trace # first add dd-trace require
+# then manually add following entry to your `composer.json` ”require” entry
+#  "opentracing/opentracing": "@dev"
+
+#  Example end result should look like:
+#  "require": {
+#   "datadog/dd-trace": "^0.2.2",
+#    "opentracing/opentracing": "@dev"
+#  }
+# Next run:
+composer update
+```
+
+### Enabling tracing
+
+#### Laravel integration
+
+To enable Laravel integration we need to configure a new Provider in `config/app.php`
+
+```php
+   'providers' => array_merge(include(base_path('modules/system/providers.php')), [
+# .....
+     'DDTrace\Integrations\LaravelProvider',
+```
+
+Now your Laravel application should start sending traces to the Datadog agent running on localhost (in default configuration). The Datadog agent must have APM enabled; see https://docs.datadoghq.com/tracing/setup/ for instructions on installing and configuring the agent.
+
+#### Symfony integration
+
+For Symfony applications, add the bundle in app/AppKernel.php:
+
+
+```php
+    $bundles = [
+        // ...
+        new DDTrace\Integrations\SymfonyBundle(),
```

Comment:
This comment is a sort of continuation of the previous one. In that case it would be the class reference and not an instance

Reply:
```suggestion
        DDTrace\Integrations\SymfonyBundle::class => ['all' => true],
```